lib/options.py

```diff
@@ -297,6 +297,21 @@ def __init__(self):
                 last position), test-run will show a list of tags
                 and stop.
                 """)
+        parser.add_argument(
+                '--env',
+                dest='show_env',
+                action='store_true',
+                default=False,
+                help="""Print environment variables, which are set by test-run.
+
```

Comment:
Please delete all empty lines, it is useless. It will be one line anyway.

Reply:
In my opinion argparse mangle my nicely formatted help messages :)

I found a solution. Please, check it out: PR #337.

Reply:
Ok, let's proceed in #337.